### DISCUSSION TITLE
Sample "Browser" not work

### DISCUSSION THREAD

**Discussion by ruipoliveira** [fief-dev/fief#370] [Q&A]
Hi guys.
I have a fief instance in my infrastructure. 
I was trying to run the example available in fief documentation, but is showing the error invalid client. Any idea? Thanks

https://docs.fief.dev/integrate/javascript/frontend/browser/

![image](https://github.com/fief-dev/fief/assets/10942886/5f206c5d-fced-4ad9-8487-3b3a1d67a9a3)

My baseURL is `{https://auth.mydomain}/{tenant}` . It's correct?

**Comment by frankie567** [comment]
Did you set your Client as a [Public](https://docs.fief.dev/configure/clients/#public-clients) one? It's a requirement for browsers integration that can't use the client secret.

**Reply by ruipoliveira** [reply]
Hi @frankie567. Yes, the client is public.

**Reply by ruipoliveira** [reply]
I think the issue is related to the client is "First-party". After create new client for same tenant it's works.
